web/src/lib/components/StallView.svelte

```diff
@@ -144,6 +145,14 @@
     });
 </script>
 
+<script context="module" lang="ts">
+  export enum AvatarSize {
```

Comment:
AvatarSize is already defined in Avatar.svelte. Please import and reuse that rather than redefining it here.